packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -119,6 +148,8 @@ export default class HostItem extends Component {
                 <InconsistentHostTimeWarning host={host} />
                 &nbsp;
                 {hasLicenseRestrictions(host) && <LicenseWarning />}
+                &nbsp;
+                {host.productBrand === 'XCP-ng' && proSupportIcon}
```

Comment:
Return early in `_getProSupportIcon` instead of testing host product brand here.